### ISSUE TITLE
Only localise links if target document exists

### ISSUE THREAD

**Issue by matthiasrohmer** [GoogleChrome/developer.chrome.com#4070]
Currently links on localized pages are prefixed with the current locale. So for example every card on the Spanish Blog overview page is prefixed with /es/ which then usually leads to a redirect, removing the /es/ locale prefix as there is no translation.

We actually know this at build time and could save the user from an unnecessary redirect, by writing the correct URL to the file.

But also this would help with static link checking, without requiring to fire up a server and waiting for HTTP requests by using tools like https://github.com/lycheeverse/lychee or https://github.com/untitaker/hyperlink#options.

This should be done before working on https://github.com/GoogleChrome/developer.chrome.com/issues/100.

**Comment by matthiasrohmer** [comment]
Hyperlink is insanely quick, taking around 2s to scan the whole project, including support for checking anchors but currently returns:

```
./hyperlink ./dist --check-anchors --sources ./site
...
Found 263559 bad links
Found 1179 bad anchors
```

Lychee also checks external links but has a slightly confusing configuration. A first test run took only 1.2s but ignored most of the links:

```
lychee --offline ./dist
...
🔍 292137 Total ✅ 3830 OK 🚫 618 Errors (HTTP:618) 💤 287689 Excluded
```

**Comment by samisharoy0** [comment]
Top 5 Model Genuine Call Girls In Noida Sector 70

**Comment by matthiasrohmer** [comment]
Won't fix, due to the pending move to the new CMS.
